src/sentry/interfaces/exception.py

```diff
@@ -795,6 +796,46 @@ def iter_tags(self):
             yield ('handled', self.handled and 'yes' or 'no')
 
 
+def uncontribute_non_stacktrace_variants(variants):
```

Comment:
More specific context around this being that it's to support a legacy unintended behavior would be helpful so that we don't think that this is actually desirable for some reason, I think?